zulip_bots/zulip_bots/bots/flock/flock.py

```diff
@@ -19,10 +19,8 @@ def find_recipient(res: str, to: str) -> str:
             return obj['id']
 
 # Returns User's ID, if not found, returns error message.
-def get_recipient_id(content: str, config: Dict[str, str]) -> str:
+def get_recipient_id(to: str, config: Dict[str, str]) -> str:
```

Comment:
In this file, why have we renamed `content` to `to` in one place and `recipient_name` in another?